test/unit/Cover/createStakingPool.js

```diff
@@ -98,10 +100,17 @@ describe('createStakingPool', function () {
         productInitializationParams,
         depositAmount,
         trancheId,
+        'ipfsDescriptionHash', // ipfsDescriptionHash
       ),
     )
       .to.emit(cover, 'StakingPoolCreated')
-      .withArgs(firstStakingPoolAddress, poolId, stakingPoolManager.address, stakingPoolImplementation);
+      .withArgs(
+        firstStakingPoolAddress,
+        poolId,
+        stakingPoolManager.address,
+        stakingPoolImplementation,
+        'ipfsDescriptionHash',
```

Comment:
same observation as in the other PR, i'd just pick a name that doesn't match the field name exactly